PWGMM/Rivet/DataModel/RivetAOs.cxx

```diff
@@ -0,0 +1,232 @@
+// Copyright 2023-2099 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+#include "RivetAOs.h"
+#include <YODA/IO.h>
+#include <TSystem.h>
+#include <TClass.h>
+#include <vector>
+#include <string>
+#include <Rivet/AnalysisHandler.hh>
+#include <Rivet/Tools/RivetPaths.hh>
+// Sigh - linter needs additional headers already included
```

Comment:
Sometimes the linter asks to include a header from identifiers are used inline in another header.  In that case, it makes little sense to include the third-level header, since the identifier is already included by the second-level header.